### DISCUSSION TITLE
shale gas vs natural gas

### DISCUSSION THREAD

**Discussion by phiphou** [electricitymaps/electricitymaps-contrib#5025] [Ideas]
Is there a way to take in account shale gas, witch has a CO2 factor higher than natural gas ? I know France import shale gas from USA, would be cool if we can distinct it from natural gas. 

![Comparison-of-GHG-emissions-from-shale-gas-and-conventional-natural-gas-with-low-and-high](https://user-images.githubusercontent.com/364418/216767721-f3bf5483-11a7-4a61-b970-051d5f2c86e6.png)

**Comment by VIKTORVAV99** [comment]
It might be possible to do this if we can find a source that splits out the share of natural gas vs shale gas. As it stands right now though we only support changing the co2intensity of the sources via the config files where we have yearly co2intensities (could technically support higher resolution if the data is available).

As this is not really a issue and more of an idea I'll also be moving this to a discussion instead.
